pages/first_page.py

```diff
@@ -28,19 +28,19 @@
     st.write("Number of inputs in table will be: {}".format(mtotal))
 
   # Stream inputs from the app. list_inputs give list of dictionaries with inputs and its metadata .
-  input_obj = User(user_id=userDataObject.user_id).app(app_id=userDataObject.app_id).inputs()  
-  all_inputs=input_obj.list_inputs()
+  input_obj = User(user_id=userDataObject.user_id).app(app_id=userDataObject.app_id).inputs()
+  all_inputs = input_obj.list_inputs()
 
-#added "data_url" which gives the url of the input.
+  #added "data_url" which gives the url of the input.
   data = []
-  for inp in all_inputs:
+  for inp in range(mtotal - 1):
```

Comment:
Any specific reason for `mtotal-1` range by default takes  `0 to mtotal-1`

Reply:
There was error showing like index out of range, found the reason which is due to less no of inputs in app. so added exception block and pushed it.